spec/project_spec.rb

```diff
@@ -27,14 +27,18 @@
   let(:error_handler) { Optimizely::RaiseErrorHandler.new }
   let(:spy_logger) { spy('logger') }
   let(:version) { Optimizely::VERSION }
-  let(:impression_log_url) { 'https://logx.optimizely.com/log/decision' }
-  let(:conversion_log_url) { 'https://logx.optimizely.com/log/event' }
+  let(:impression_log_url) { 'https://logx.optimizely.com/v1/events' }
+  let(:conversion_log_url) { 'https://logx.optimizely.com/v1/events' }
   let(:project_instance) { Optimizely::Project.new(config_body_JSON, nil, spy_logger, error_handler) }
   let(:time_now) { Time.now }
   let(:post_headers) { { 'Content-Type' => 'application/json' } }
 
   it 'has a version number' do
-    expect(Optimizely::VERSION).not_to be nil
+    expect(Optimizely::VERSION).not_to be_nil
```

Comment:
Are we sure we need engine name and version not to be nil? 
The new endpoint api does not say 'required' with client_name and client_version
https://developers.optimizely.com/x/events/api/index.html

Reply:
By the way, I thought about it again and I see no harm in ensuring that when ever a payload is generated using this SDK, it always has a client name and version

Reply:
updated to use proper syntex, not added it